test/run.lua

```diff
@@ -114,7 +114,9 @@ local function approximateLine(line)
 	if string.match(line, "Logs")                then return 120 end
 	if string.match(line, "Temporary")           then return 120 end
 	if string.match(line, "Directory")           then return 160 end
-	if string.match(line, "seconds")             then return   5 end
+	if string.match(line, "less than one")       then return  14 end
+	if string.match(line, "second")              then return  14 end
+	if string.match(line, "seconds")             then return  14 end
```

Comment:
`seconds` is unnecessary, as `second` is a substring. `less than one` seems to be unnecessary as well, as it will have `second` in the line.